spec/anagrams_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'anagrams'
+
+describe Anagrams do
+  describe '.group_anagrams' do
+    it 'Returns anagrams from a file: ' do
+      anagram = Anagrams.new('AnagramFile.txt')
+      expected_anagrams = [%w[spate tapes pates septa peats tepas paste],
+                           %w[cuprites pictures crepitus piecrust], %w[undress sunders]]
+      expect(anagram.group_anagrams).to eql expected_anagrams
+    end
+
+    it 'Returns anagrams from array: ' do
```

Comment:
@zzokki81 this outputs quite unreadable and unusable print out:

```
$ rspec -fd

Anagrams
  .group_anagrams
    Returns anagrams from a file:
    Returns anagrams from array:
```

`describe` and `it` descriptions should resemble sentences as much as possible.

Beside that... 1) You need to add specs for other public methods in Anagrams class. 2) Whenever I report a "bug", like "ABC and abc are not detected as anagrams" you should add a failing test and then fix the code. Please add specs for these edge cases I reported earlier. That way test suite grows and covers more cases over time.